osu.Game.Rulesets.Osu/UI/OsuTouchInputMapper.cs

```diff
@@ -97,26 +97,32 @@ private void updatePositionTracking(TrackedTouch newTouch)
                 return;
             }
 
-            // ..or if the current position tracking touch was not a direct touch (this one is debatable and may be change in the future, but it's the simplest way to handle)
-            if (!positionTrackingTouch.DirectTouch)
+            // ..or if the current position tracking touch was not a direct touch (and didn't travel across the screen too far).
+            if (!positionTrackingTouch.DirectTouch && positionTrackingTouch.DistanceTravelled < 200)
```

Comment:
while initially testing this, i had no idea why the tracking would *sometimes* work as expected, while sometimes it would switch to my tapping hand.

turns out i was just not dragging my tracking finger far enough before tapping with my other hand.

https://user-images.githubusercontent.com/36758269/225689628-2b5c7a7d-98a2-4eb7-8a19-534245670c57.mp4

i think that this distance requirement should be lowered to maybe somewhere between 5 and 30 (haven't experimented changing this myself yet), but honestly i have no idea why this logic should exist in the first place. shouldn't the first touch always keep tracking while held?

Reply:
It's not the direction I want to take sorry. It severely deprives the user feedback loop for non-offscreen-tapping gameplay.

Please open a discussion if you disagree – this PR is not the place to attempt to change how the base touch input works.